src/components/AgentInfo.tsx

```diff
@@ -12,7 +12,7 @@ export default function AgentInfo({ agent }: IAgentInfo) {
   return (
     <div className="d-flex flex-column">
       <span className="m-0 large h4 lh-sm">
-        {agent.lastName}, {agent.firstName}
+        {agent.firstName ? (agent.lastName, agent.firstName) : agent.lastName}
```

Comment:
This expression in parens `(agent.lastName, agent.firstName)` isn't returning the `Last, First` string as expected. Instead it's just rendering a single name for all agents:

<img width="300" alt="image" src="https://github.com/ProjectProtocol/project-protocol-web/assets/6488787/375316e3-bc34-4715-bd34-e74dff212eb0">

```suggestion
        {agent.firstName ? `${agent.lastName}, ${agent.firstName}` : agent.lastName}
```

Reply:
Yikes, big oversight!